### PR TITLE
added text utilities & lists

### PR DESCRIPTION
**Pull request by kedark152**
### #feat: added text utilities & lists
1. **Text Utilities** consists of Headings, Small text, gray text & Text with alignments.
2. **Lists** consists of Spaced lists & stacked lists.

### CODE REVIEW

File: stylesheets/components/text-utils.css [hunk 1]
```diff
@@ -0,0 +1,19 @@
+.grey-text{
+    color: grey;
```
Comment (omgaikwad) [inline stylesheets/components/text-utils.css]: you can add "grey" color in  variable and use variable instead

**Review by omgaikwad — COMMENTED** [review]
Everything looks good!

File: stylesheets/components/text-utils.css (hunk 1 above)
Comment (kedark152) [reply stylesheets/components/text-utils.css]: Updated!! please check & approve.

Comment (omgaikwad) [reply stylesheets/components/text-utils.css]: You can merge

**Review by bhargaviRdy — APPROVED** [review]
Looks good

**Review by Raghav888 — APPROVED** [review]

**Pull request merged**